### DISCUSSION TITLE
FAST/SLOW IIDX or OD under combo Quaver analog in osu!

### DISCUSSION THREAD

**Discussion by Dopaminos** [ppy/osu#14567] [Ideas] [answered]
Hi, i'd wanted to suggest one thing, for what any vsrg-player could thank you. I'm not very good player, but after playing other games like stepmania (etterna, quaver) and iidx i feel what osu needs one thing. When i play osu!mania i can see what i got: 320, 300, 200... but i can't know why i got this: because i was too slow or too fast? And because of that i've wished this thing would be added in osu.
Existing right now OD placement is bad for osu!mania since usually, you are looking on grade (320 and etc) or downer and you can't turn your head down to get this information and if you'll try - you can easily drop accuracy
### Quaver OD bar under your grade (marv, perfect and etc)
![browser_0tpuKDFbh0](https://user-images.githubusercontent.com/42616718/131254162-1320b8c4-c1f8-412c-8eda-1bf67790893f.png)

**Comment by bdach** [comment, marked as answer]
See https://github.com/ppy/osu/pull/7310.

Also I'm pretty sure you should already be able to drag the hit error meter wherever you please on the playfield using the skin editor.

**Reply by Dopaminos** [reply]
i can't mark it as answer iuserthguiotsehgsuoierthjg;nbi
